common/teacmd/editor.go

```diff
@@ -14,16 +14,35 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	"github.com/rotisserie/eris"
+	"golang.org/x/mod/modfile"
 
 	"pkg.world.dev/world-cli/common/globalconfig"
 )
 
 const (
 	TargetEditorDir = ".editor"
 
-	latestReleaseURL             = "https://api.github.com/repos/Argus-Labs/cardinal-editor/releases/latest"
-	httpTimeout                  = 2 * time.Second
+	releaseURL       = "https://api.github.com/repos/Argus-Labs/cardinal-editor/releases"
+	latestReleaseURL = releaseURL + "/latest"
+
+	httpTimeout = 2 * time.Second
+
 	cardinalProjectIDPlaceholder = "__CARDINAL_PROJECT_ID__"
+
+	cardinalGomodPath = "cardinal/go.mod"
+
+	cardinalPkgPath = "pkg.world.dev/world-engine/cardinal"
+)
+
+var (
+	// Cardinal : Cardinal Editor version map
+	// TODO this version map need to store in somewhere that can be accessed online
+	cardinalVersionMap = map[string]string{
+		"v1.2.2-beta": "v0.1.0",
+		"v1.2.3-beta": "v0.1.0",
+		"v1.2.4-beta": "v0.2.0",
```

Comment:
bug fixes in v1.2.5-beta & v0.3.0
```suggestion
		"v1.2.4-beta": "v0.3.0",
		"v1.2.5-beta": "v0.3.0",
```